src/config/logging.rst

```diff
@@ -137,3 +137,13 @@ Logging options
 
             [log]
             syslog_facility = local2
+
+    .. note::
+        CouchDB's ``syslog`` only knows how to use UDP logging. Please ensure that your
+        ``syslog`` server has UDP logging enabled.
```

Comment:
Well written. Let's add a note that `syslog_host` must be specified for the logging to work.  `local2` is already a default, so technically not needed to specified.

Reply:
50% of these two sentence did you contribute ;-)
The `local2` example was in the docs before. Should we keep it or use another value?